### PR TITLE
 Add Python Event Hubs 5.2.0 (September) release notes.

### PR DESCRIPTION
**Pull request by KieranBrantnerMagee**
Conn str SAS token support, easy release this month :)

### CODE REVIEW

File: releases/2020-09/python.md [hunk 1]
```diff
@@ -32,6 +33,7 @@ pip install azure-eventgrid --pre
 pip install azure-keyvault-certificates
 pip install azure-search-documents --pre
 pip install azure-appconfiguration
+pip install azure-eventhubs
```
Comment (xiangyan99) [inline releases/2020-09/python.md:36]: Do you need --pre?

Comment (KieranBrantnerMagee) [reply releases/2020-09/python.md:36]: Nope.  (ServiceBus still in pre, this is a minor-version-# release.)

File: releases/2020-09/python.md [hunk 2]
```diff
@@ -21,6 +21,7 @@ The Azure SDK team is pleased to make available the September 2020 client librar
 
 - Event Grid
 - Search
+- Event Hubs
```
Comment (xiangyan99) [inline releases/2020-09/python.md:24]: Then should we move this to "Updates"?

Comment (KieranBrantnerMagee) [reply releases/2020-09/python.md:24]: Ah; nice catch; clearly SB muscle memory too strong :)

**Review by xiangyan99 — APPROVED** [review]

**Pull request merged**